int64 numA
int64 numB
---
int64 sum
